src/main/scala/edu/colorado/plv/cuanto/scoot/jimple/AddExpr.scala

```diff
@@ -0,0 +1,16 @@
+package edu.colorado.plv.cuanto.scoot.jimple
+
+/**
+  * Created by Jared on 6/21/2017.
+  */
+class AddExpr(dt: soot.jimple.AddExpr) extends BinopExpr {
+  def op1: Value = dt.getOp1()
```

Comment:
In the spirit of DRY, we drop `op1`.

So this becomes `class AddExpr private[jimple] (private[this] val dt: soot.jimple.AddExpr) extends BinopExpr`.